docs/guide/testing.md

```diff
@@ -1,14 +1,15 @@
 # Testing
-
 The tests are done with Jest and Jasmine. The same test suite should
 pass on both of them because the library might be used
 [server-side](server-side-installation) or in a browser, so you have
 to be sure that both environments are fine.
 
-* **`npm run test`** - runs the linter and all tests 
+* **`npm run test`** - runs the linter and all tests
 * **`npm run test:unit`** - runs unit tests
-* **`npm run test:browser` -** runs karma test runner in browsers,
-prints the results in a console
+* **`npm run test:browser`** - runs tests in **karma** once and closes all open browsers
+* **`npm run test:browser.debug`** - runs test in **karma** only in Chrome until you exit the process. It watches changes in `src` and `test` directories and rebuilds them automatically.
+
+An additional flag for **karma** tests: `-spec`. For example: `npm run test:browser.debug -- --spec=matrix.spec.ts`
```

Comment:
`-spec` is to run a specific `spec` file or a test suite. It's not always necessary or desireable to wait for all the tests to run.